apps/mobile/src/components/ContextSwitchMenu/ContextSwitchMenu.js

```diff
@@ -0,0 +1,137 @@
+import React from 'react'
+import { Animated, TouchableOpacity, Text, StyleSheet, View, FlatList } from 'react-native'
+import FastImage from 'react-native-fast-image'
+import { Globe, Plus, CircleHelp } from 'lucide-react-native'
+import { map, sortBy } from 'lodash/fp'
+import useCurrentUser from '@hylo/hooks/useCurrentUser'
+import useCurrentGroup from '@hylo/hooks/useCurrentGroup'
+import { PUBLIC_GROUP } from '@hylo/presenters/GroupPresenter'
+import useChangeToGroup from 'hooks/useChangeToGroup'
+
+export default function ContextSwitchMenu () {
+  const [{ currentUser }] = useCurrentUser()
+  const [{ currentGroup }] = useCurrentGroup()
+  const changeToGroup = useChangeToGroup()
+  const myGroups = [PUBLIC_GROUP].concat(sortBy('name', map(m => m.group, currentUser.memberships)))
+
+  return (
+    <Animated.View className='flex-col h-full bg-theme-background z-50 items-center py-2 px-3'>
+      {/* FlatList used instead of FlashList because of strict-sizing requirements of FlashList */}
+      <FlatList
+        data={myGroups}
+        keyExtractor={(item) => item.id}
+        renderItem={({ item }) => (
+          <NavRow
+            changeToGroup={changeToGroup}
+            item={item}
+            currentGroupSlug={currentGroup?.slug}
+          />
+        )}
+        showsVerticalScrollIndicator={false}
+      />
+      <View className='w-full mt-auto bg-theme-background pt-4'>
+        <TouchableOpacity 
+          onPress={() => {}} // TODO redesign: Needs to open some creation dialog...
+          style={styles.rowTouchable}
+          activeOpacity={0.7}
+        >
+          <View
+            className={`
+                bg-primary relative flex flex-col text-primary-foreground items-center justify-center
+                w-14 h-14 min-h-10 rounded-lg drop-shadow-md opacity-60 scale-90
+            `}
```

Comment:
This is one way to handle the long className lists. I for now am mostly looking the other way and not going to worry about the inevitably very long lines here with the introduction of NativeWind, but wanted to highlight this option.